docs/open_source/testset_generation/testset_generation/index.md

```diff
@@ -279,6 +279,8 @@ from giskard.rag import QATestset
 loaded_testset = QATestset.load("my_testset.jsonl")
 ```
 
+You can push your generated test set to the Hugging Face Hub or load an existing dataset from it using our integration. This allows you to share and reuse datasets easily. For detailed instructions, refer to the [Hugging Face Integration Documentation](https://docs.giskard.ai/en/latest/integrations/huggingface/QATestset.html).
```

Comment:
You can also share the code or the API reference.

Reply:
I added the API reference